index.html

```diff
@@ -7,6 +7,7 @@
   <div id="intro">
     <h1>Bienvenue sur l’Aide officielle The Ebook Alternative</h1>
     <p>Retrouvez toutes les informations nécessaires sur nos produits et ceux de <a href="https://www.tea-ebook.com/societe/#clients" title="TEA - the ebook alternative">nos partenaires</a> liés à l’achat et la lecture de livres numériques sur tous supports.</p>
+    <p>Pour toute question sur une liseuse Vivlio, consultez <a href="https://help.vivlio.fr" title="Aide en ligne Vivlio">l'Aide en ligne Vivlio</a></p>  
```

Comment:
Je suggère d'ajouter un point à la fin de la phrase : 

```suggestion
    <p>Pour toute question sur une liseuse Vivlio, consultez <a href="https://help.vivlio.fr" title="Aide en ligne Vivlio">l'Aide en ligne Vivlio</a>.</p>  
```